### ISSUE TITLE
Pagination is broken if there is only one page

### ISSUE THREAD

**Issue by kennyx46** [themesberg/flowbite-react#1048]
- [X] I have searched the [Issues](https://github.com/themesberg/flowbite-react/issues) to see if this bug has already been reported
- [X] I have tested the latest version

## Steps to reproduce

1. Render pagination component with this props:
```
<Pagination 
  currentPage={1} 
  totalPages={1} 
  showIcons
  nextLabel=""
  previousLabel=""
/>
```

## Current behavior

There is no `1` between the arrows, just two buttons (previous and next) 
<img width="138" alt="Screenshot 2023-10-06 at 11 33 51" src="https://github.com/themesberg/flowbite-react/assets/3941650/4fb13454-b469-434e-8e3d-df015c12fccf">

## Expected behavior

`1` is shown between the arrows, similarly to the case when we have 2 and more pages.
<img width="260" alt="image" src="https://github.com/themesberg/flowbite-react/assets/3941650/0687ffc3-2b96-4058-a64a-d846499a26c0">

## Context

I would like to show the `Pagination` component with only one page when user has a small amount of data.

The problem is in `range` function, where this check is contained
```
if (start >= end) {
    return [];
}
```
I would assume it should be
```
if (start > end) {
    return [];
}
```
but not sure whether this breaks the expected behaviour from your side. If you are fine I can create the PR.

Thanks,
Sergii

**Comment by MateoWartelle** [comment]
I think maybe [this](https://github.com/themesberg/flowbite-react/blob/main/src/components/Pagination/Pagination.tsx#L93) needs to be currentPage < 1 also

For this 

``` <Pagination 
  currentPage={0} 
  totalPages={0} 
  showIcons
  nextLabel=""
  previousLabel=""
/> ```

<img width="147" alt="pagination-0" src="https://github.com/themesberg/flowbite-react/assets/27743176/07fdbfbf-3c2a-48d2-8420-f8e089edc0ba">

we wouldn't want them to be able to click previous if no pages are available

**Comment by SutuSebastian** [comment]
> we wouldn't want them to be able to click previous if no pages are available

Agreed, but this doesn't mean we do not show the pages, even if its just one.

**Comment by Mukilan1600** [comment]
@tulup-conner I've raised a PR to fix this

**Comment by crudobaker** [comment]
I have the same issue. Just waiting for the fix.
Thanks!

**Comment by yamin-H** [comment]
Hi, can I work on this if it's not actively maintained?

**Comment by DevLikhith5** [comment]
I'd like to fix this bug. The issue is that Pagination doesn't handle the single-page edge case — it should either hide or disable navigation when there's only one page. Planning to open a PR with the fix.

**Comment by JessenReinhart** [comment]
I've raised a PR for this
